scripts/train.py

```diff
@@ -0,0 +1,41 @@
+import os
+import click
+import pickle
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from sklearn.neural_network import MLPClassifier
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
+from pkspace.utils import trainer
+
+
+@click.command()
+@click.option('--PKSpace', 'dataset_mode', flag_value='PKSpace',
+              default=True)
+@click.option('--PKLot', 'dataset_mode', flag_value='PKLot')
+@click.option('--MLP', 'method', flag_value='MLP',
+              default=True, help='Method to be used for prediction')
```

Comment:
@rstevanak I also think this can be multiple choice argument. It will be more easily extensible in the future. When we add more models. Also description should be changed probably to something like `Model which will be trained` or something like that.